kaiax/gasless/interface.go

```diff
@@ -23,7 +23,7 @@ import (
 
 //go:generate mockgen -destination=./mock/module.go -package=mock github.com/kaiachain/kaia/kaiax/gasless GaslessModule
 type GaslessModule interface {
-	kaiax.BaseModule
```

Comment:
I also think BaseModule might not be mandatory, but let's just keep the small junk at this point. Let's rethink when we revise the cn.New() function.

Reply:
fixed at https://github.com/kaiachain/kaia/pull/286/commits/3bfc554cffc0fecdd16f50e7a5bdc386ffcd20f3